os/fs/smartfs/Kconfig

```diff
@@ -73,22 +73,14 @@ bool "Dynamic Header"
 	---help---
 		Using Modified Used Byte Method to Reduce Sector Relocation 
 
-config SMARTFS_JOURNALING
-        bool "Enable filesystem journaling for smartfs"
-        default n
-        ---help---
-                Enables logging of write operations on filesystem, so that in
-                case of power failure the filesystem can be brought to a stable
-                state with minimal data loss.
-                Enabling journaling will increase the delay in filesystem
-                operations, because before doing any fs operation, journaling
-                will first log it in journal area. The journal area will also
-                take up space in the flash, leaving lesser space for filesystem.
-                The space reserved for journal area is proportional to max number
-                of file descriptors allowed and sector size defined. Thus, to
-                minimize the area reserved for journaling, it is advised to keep
-                sector size small.
-
+config SMARTFS_SECTOR_RECOVERY
```

Comment:
Should the option for recovery be provided under journaling?
Or will we continue to allow it as a separate feature?

Reply:
Actually Recovery should start from fs layer, so I think let it work separetely for now.